### ISSUE TITLE
Unexpected max serialisation length

### ISSUE THREAD

**Issue by benguela** [commschamp/commsdsl#4]
I defined the following list

```
            <list name="VideoBlocks">
                <element>
                    <data name="VideoBlock" length="64"/>
                </element>
                <countPrefix>
                    <int name="VideoBlockCounter" type="uint8" />
                </countPrefix>
            </list>
```
I get the following error on compile

```
`error: static assertion failed: Unexpected max serialisation length
  900 |     static_assert(MsgMaxLen == 16352U, "Unexpected max serialisation length");`
```
If I remove the length property from the data field it compiles fine but I can't use this as each VideoBlock is a fixed length (64 bytes) and I have fields after the VideoBlocks in my protocol.

Am I specifying my schema incorrectly for this case?

many thanks.

**Comment by arobenko** [comment]
Hi Pablo, it's a known issue for me at this moment. It's fixed on develop
branch. Please try it. I'll do the patch release soon.

**Comment by arobenko** [comment]
Should be fixed now on "develop" branch.

**Comment by benguela** [comment]
tested and working.

thanks

**Comment by arobenko** [comment]
Fixed in [v3.4.1](https://github.com/arobenko/commsdsl/releases/tag/v3.4.1)
